src/ResourceManager/Sql/Commands.Sql/Failover Group/Cmdlet/NewAzureSqlFailoverGroup.cs

```diff
@@ -78,15 +77,14 @@ public class NewAzureSqlFailoverGroup : AzureSqlFailoverGroupCmdletBase
         public AllowReadOnlyFailoverToPrimary AllowReadOnlyFailoverToPrimary { get; set; }
 
         /// <summary>
-        /// Gets or sets the tag associated with the Azure Sql Failover Group
+        /// Gets or sets the tag associated with the Azure SQL Database Failover Group
         /// </summary>
         [Parameter(Mandatory = false,
-            HelpMessage = "The tag to associate with the Azure Sql Failover Group")]
-        [Alias("Tag")]
+            HelpMessage = "The tag to associate with the Azure SQL Database Failover Group")]
         public Hashtable Tag { get; set; }
 
         /// <summary>
-        /// Overriding to add warning message
+        /// Execute the cmdlet
         /// </summary>
         public override void ExecuteCmdlet()
```

Comment:
This method isn't needed if all you are doing is calling base.ExecuteCmdlet().  Please remove it.